crates/wasmcloud-host/src/host.rs

```diff
@@ -521,6 +521,47 @@ impl Host {
             .collect())
     }
 
+    /// Retrieves human-friendly identity information for a provider
+    /// Return value is `(image_reference, name, revision)`
+    pub async fn get_actor_identity(&self, id: &str) -> Result<(Option<String>, String, i32)> {
+        self.ensure_started()?;
+        let hc = HostController::from_hostlocal_registry(&self.id);
+        if let Some(actor) = hc
+            .send(GetRunningActor {
+                actor_id: id.to_string(),
+            })
+            .await?
+        {
+            let identity = actor.send(GetIdentity {}).await?;
+            Ok((identity.image_ref, identity.name, identity.revision))
+        } else {
+            Err("Actor is not running on this host".into())
+        }
+    }
+
+    /// Retrieves human-friendly identity information for a provider
+    /// Return value is `(image_reference, name, revision)`
+    pub async fn get_provider_identity(
+        &self,
+        id: &str,
```

Comment:
Explicitly call out in the rustdoc comment that this `id` parameter is the public key of the provider.

Reply:
Good call. I'll rename it to `provider_id` as well to make it immediately clear